chainer/link_hooks/weight_standardization.py

```diff
@@ -0,0 +1,80 @@
+from chainer.functions.normalization._standardize import _standardize
```

Comment:
Could you import a module, not a function?
```suggestion
from chainer.functions.normalization import _standardize
```
and use `_standardize._standardize` later.